src/state/api.js

```diff
@@ -29,7 +30,11 @@ export async function getProbeData(params, token) {
   }).then(async (response) => {
     // catch fetch error responses and surface them into
     // the frontend.
-    if (response.status >= 400 && response.status < 600) {
+    if (response.status === 404) {
+      const e = new Error(FETCH_ERROR_MESSAGES.code404);
+      e.moreInformation = "You may try changing the probe's current data filters.";
```

Comment:
There's a Prettier error on this line. It looks like it wants the line to be shorter.

In both cases, simply running `npm run format` should do the formatting for you.